certbot/log.py

```diff
@@ -23,36 +24,82 @@
 
 
 def pre_arg_parse_setup():
-    """Ensures fatal exceptions are logged and reported to the user."""
-    sys.excepthook = functools.partial(except_hook, config=None)
+    """Setup logging before command line arguments are parsed.
+
+    Terminal logging is setup using
```

Comment:
Instead I added a more general description to the top of the module and made this docstring slightly more specific. I think we need to reference `post_arg_parse_setup` here though because the two functions work together closely and the location of the log file depends on whether or not `post_arg_parse_setup` is called.